README.md

```diff
@@ -439,6 +439,7 @@ Discover the full list of Nuxt modules on https://modules.nuxtjs.org
 - [hex-digital/nuxtpress](https://github.com/hex-digital/nuxtpress) - A modern Nuxt + headless WordPress starter kit; dockerised, SEO-friendly and with development and testing tools pre-integrated.
 - [mmoollllee/nuxt-starter-template](https://github.com/mmoollllee/nuxt-starter-template) – Template for basic static site generated marketing website.
 - [vinayakkulkarni/module-template-ava](https://github.com/vinayakkulkarni/module-template-ava) - Starter template for Nuxt.js Modules using Ava instead of Jest.
+- [t-miller/nuxt.js-starter-dimension](https://github.com/t-miller/nuxt.js-starter-dimension) - Static portfolio site template using Nuxt + Sass
```

Comment:
Could be more explicit to rename to `nuxt-portfolio-starter-dimension`. 
What do you think about it ?

Reply:
Furthermore, your commit need to respect the conventional commits. 
You can see a quick explanation on : https://github.com/zeke/semantic-pull-requests#configuration